src/parseTestRecord.py

```diff
@@ -35,7 +35,7 @@
        r'// Use of this source code is governed by a BSD-style license that can be[\r\n]{1,2}' +
        r'// found in the LICENSE file\.' +
        r'|' +
-       r'// See LICENSE or https://github\.com/tc39/test262/blob/master/LICENSE' +
+       r'// See LICENSE or https://github\.com/tc39/test262/blob/HEAD/LICENSE' +
```

Comment:
```suggestion
       r'// See LICENSE or https://github\.com/tc39/test262/blob/(master|HEAD)/LICENSE' +
```

This won't mess up the matching group, I believe so.

I've created a very quick test here: https://gist.github.com/leobalter/ccee8b1d9dda74cfa4a1d21664a5bbdd

Out of curiosity, this regexp part exists to capture only two test files in Test262:

```
test/built-ins/Array/prototype/concat/S15.4.4.4_A3_T2.js
test/built-ins/Array/prototype/concat/S15.4.4.4_A3_T3.js 
```